var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -113,6 +113,14 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage):
 
     # https://gcc.gnu.org/install/prerequisites.html
     depends_on('gmp@4.3.2:')
+    # mawk is not sufficient for go support
+    depends_on('gawk@3.1.5:')
+    depends_on('texinfo@4.7:', type='build')
```

Comment:
Isn't this only required on `master`? From https://gcc.gnu.org/install/prerequisites.html:

> Necessary to build GCC documentation during development because the generated output files are not included in the repository. They are included in releases.

Reply:
That's what the website says, but we had repeated failures building the gcc rse-ops containers unless it's included.  I think some piece of it snuck in as a requirement for one of the sub-configurations even on release versions.  At least the basics of it are almost always there, either because of the base system or one of gcc's dependencies, so it only becomes noticeable when building inside a very minimal stripped ubuntu container under particular circumstances, but it gets used.  Have a look at the log for [this gcc-8 build](https://pipelines.actions.githubusercontent.com/rzVSkgzD01NrGHpMB9JsMS8MQAldYURCmBwbfMoIp1kCRgpXso/_apis/pipelines/1/runs/4623/signedlogcontent/26?urlExpires=2022-03-23T14%3A52%3A30.1581949Z&urlSigningMethod=HMACV1&urlSignature=pi038VmopNrFVt0NXlyqHp41ryCtcsR5z2TRd4Bg9ag%3D) over there and search for timestamp 2022-03-04T12:08:29.8344233Z where it's getting run to generate the gcc docs during a release build.

Reply:
Ah, and there's another one in there that I missed, libtool is unconditionally required as well. I'll add that momentarily.

Reply:
Looks like all is green, what say ye @alalazo?